### DISCUSSION TITLE
Blender Launcher v1.15.2

### DISCUSSION THREAD

**Discussion by Victor-IX** [Victor-IX/Blender-Launcher-V2#1] [Announcements]
# What's Changed
First version after Forking from the original project. 
I will keep updating and improving this software, don't hesitate to give me feedback and create pull requests if you want to add your own feature in this repo!
You can also join my [Discord](https://discord.gg/3jrTZFJkTd) if you want to follow the development

## ⚠Info:
- Some Blender Websites are still not fully working after the recent DDoS attack you may not be able to download some of the Blender builds
- It can take some time to get all the Blenders versions so be patient when the program is checking for new versions
- Only the Windows version is available for now, I need to set up a Linux environment to build the Linux version
- Expect bugs, I was not able to test everything 

## 🛠️Fixed:
- Fix Experimental and Daily not being loaded
## ⚙️Other:
- Raise the stable pull version to Blender 3.0 instead of Blender 2.79
_This is temporary, I had to do this because Blender is limiting the number of requests per minute due to the recent DDoS attack, otherwise, pulling all the versions is taking a very long time and blocking the Daily and Experimental versions from being pulled_
- Remove the /patch build from the experimental section

<hr /><em>This discussion was created from the release <a href='https://github.com/Victor-IX/Blender-Launcher/releases/tag/v1.15.2'>Blender Launcher v1.15.2</a>.</em>

**Comment by lonely814** [comment]
Can you add language options, such as Chinese?
